disperser/batcher/batcher.go

```diff
@@ -455,23 +445,41 @@ func (b *Batcher) HandleSingleBatch(ctx context.Context) error {
 			signerCount[quorumID] = 0
 		}
 		for opID := range opState {
-			if _, ok := aggSig.SignerMap[opID]; ok {
+			if _, ok := quorumAttestation.SignerMap[opID]; ok {
 				signerCount[quorumID]++
 			}
 		}
 	}
-	b.Metrics.UpdateAttestation(operatorCount, signerCount, aggSig.QuorumResults)
-	for _, quorumResult := range aggSig.QuorumResults {
+	b.Metrics.UpdateAttestation(operatorCount, signerCount, quorumAttestation.QuorumResults)
+	for _, quorumResult := range quorumAttestation.QuorumResults {
 		log.Info("Aggregated quorum result", "quorumID", quorumResult.QuorumID, "percentSigned", quorumResult.PercentSigned)
 	}
 
-	numPassed := numBlobsAttested(aggSig.QuorumResults, batch.BlobHeaders)
+	numPassed, numPassedByQuorum := numBlobsAttestedByQuorum(quorumAttestation.QuorumResults, batch.BlobHeaders)
 	// TODO(mooselumph): Determine whether to confirm the batch based on the number of successes
 	if numPassed == 0 {
 		_ = b.handleFailure(ctx, batch.BlobMetadata, FailNoSignatures)
 		return errors.New("HandleSingleBatch: no blobs received sufficient signatures")
 	}
 
+	nonEmptyQuorums := []core.QuorumID{}
+	for quorumID, count := range numPassedByQuorum {
+		log.Info("Blobs attested by quorum", "quorumID", quorumID, "count", count)
+		if count > 0 {
```

Comment:
Should this filter out quorums that have a signing rate below confirmation threshold?

Reply:
This should be already filtering out quorums that have a signing rate below the confirmation threshold. 
`count` here in `numPassedByQuorum` is the number of blobs that have signing rate above the confirmation threshold, and we're filtering out quorums with count == 0

Reply:
Since all blobs are in the same batch, they have the same signing rate for a given quorum. It looks this doesn't need to count num blobs (it's going to be either 0 or Num-Blobs), just whether the quorum collected sufficient signing rate

Reply:
Does this make sense?

Reply:
Sure, what you're suggesting is that we can make this logic simpler by simply comparing the quorum result vs. the confirmation threshold required at each quorum which is hardcoded in contracts. Correct?
But then the batcher needs to be aware of those thresholds and make an assumption that all blobs will be dispersed using the same thresholds for each quorum. 
Comparing the security params for each blobs with the quorums results allows the batcher to just use the security params set at each blob and filter quorums more generally.

Reply:
I think numBlobsAttestedByQuorum may just return a set of quorumIDs that have collected sufficient signatures. It's semantically more clear but cutting away the count since it's 0 or all.

Reply:
Sure, updated